docker/run.sh

```diff
@@ -126,15 +130,15 @@ rm "$filename".manifest
 echo "exporting event table content"
 
 
-EVENTS_FILE=events_$(date +"%Y-%m-%dT%H-%M-%S").json
+EVENTS_FILE=events_$PERIOD_DATE.json
 
 PGPASSWORD=$DB_PASSWORD psql "sslmode=verify-ca sslrootcert=/root/.postgresql/root.crt sslcert=/root/.postgresql/postgresql.crt sslkey=/tmp/client-key.pem hostaddr=$DB_HOST port=$DB_PORT user=$DB_USERNAME dbname=rm" \
--c "\copy (SELECT row_to_json(t) FROM (SELECT * FROM casev2.event where event_type!='CASE_CREATED' and event_type!='UAC_UPDATED' and event_type!='SAMPLE_LOADED' and event_type!='RM_UAC_CREATED' and event_type!='PRINT_CASE_SELECTED') t) To '$EVENTS_FILE';"
+-c "\copy (SELECT row_to_json(t) FROM (SELECT * FROM casev2.event where event_type!='CASE_CREATED' and event_type!='UAC_UPDATED' and event_type!='SAMPLE_LOADED' and event_type!='RM_UAC_CREATED' and event_type!='PRINT_CASE_SELECTED' and event_date >= '$START_OF_PERIOD' and event_date < '$END_OF_PERIOD') t) To '$EVENTS_FILE';"
```

Comment:
I don't think this can use the `event_date`, that's the timestamp we receive on the event. I think it needs to be the `rm_event_processed` date that is filtered on here.

Reply:
👍 I did have a think about which of the time stamps to use but chose the wrong one! I'll update the query.